inspirehep/modules/records/jsonschemas/records/hep.json

```diff
@@ -684,7 +684,44 @@
         },
         "references": {
             "items": {
-                "$ref": "elements/reference.json"
+                "properties": {
+                    "curated_relation": {
+                        "type": "boolean"
+                    },
+                    "raw_refs": {
+                        "items": {
+                            "properties": {
+                                "position": {
+                                    "description": "The position of the reference as it appears on the document",
+                                    "type": "string"
+                                },
+                                "schema": {
+                                    "description": "E.g. text, or Elsevier XML, or refexract",
+                                    "type": "string"
+                                },
+                                "source": {
+                                    "type": "string"
+                                },
```

Comment:
Source is different than schema, because e.g. we could have an XML directly from Elsevier, but also a ref-extract coming from the same Elsevier PDF.